cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/monitor/sampling/KafkaSampleStore.java

```diff
@@ -260,9 +265,12 @@ private void maybeIncreaseTopicReplicationFactor(ZkUtils zkUtils,
       brokersByRack.get(rack).add(bm.id());
       rackByBroker.put(bm.id(), rack);
     }
+
+    boolean skipRackAwarenessCheck = false;
```

Comment:
Would it be possible to make `skipSampleStoreTopicRackAwarenessCheck` a configuration? We can keep the default as `true` (and throw an exception), but if the user intentionally would like to skip this check, s/he can set this config to skip this check.

Reply:
I believe you mean default value to be `false`.

Reply:
Correct, I meant to say `false`.